src/pages/learn.js

```diff
@@ -49,6 +49,44 @@ const Article = styled('article')`
     padding: ${size.medium};
 `;
 
+const FEATURED_ARTICLE_URLS = [
```

Comment:
Is this to replace the build-time prop passed in `gatsby-node.js`?

Reply:
Wow I did not realize that was there. I can probably just update the article list here rather than doing the new filtering

Reply:
Sounds good! Yep that is what I would suggest :)